### DISCUSSION TITLE
How to merge topic model with original dataset

### DISCUSSION THREAD

**Discussion by joshedOpenAi** [MaartenGr/BERTopic#1953] [Q&A]
Hi Looking for best practices to merge topic model data with the original dataset
Example:
Lets say I have a dataset
data = {
    'Feedback': ['Great product', 'Needs improvement', 'Love it', 'Not satisfied'],
    'UserClass': ['Promoter', 'Detractor', 'Promoter', 'Passive'],
    'Date': ['2022-01-01', '2022-01-02', '2022-01-03', '2022-01-04']
}
After topic modeling how to I map the topic documents back to the original data set to slice data based on Userclass in this dataframe

**Comment by MaartenGr** [comment]
The output of BERTopic, generally a variable named `topics`, are in the same order as your input documents. So you can use that order to map the topics back to your original dataset.
